src/helpers/getMovieUtils.ts

```diff
@@ -18,4 +18,9 @@ const getMovieDetails = async (id: number): Promise<MovieDetailsData> => {
 	return response.json() as Promise<MovieDetailsData>;
 };
 
-export { getMoviesByName, getMovieDetails };
\ No newline at end of file
+const getMovieProviders = async (id: number): Promise<MovieProviders> => {
```

Comment:
I know it is fairly obvious, but a comment would be good for all util functions.